doc/user_guide/run.rst

```diff
@@ -7,9 +7,10 @@ Invoking Pylint
 
 Pylint is meant to be called from the command line. The usage is ::
 
-   pylint [options] module_or_package
+   pylint [options] modules_or_packages
```

Comment:
I had tried to do that by making the terms plural.  Are you suggesting something else?